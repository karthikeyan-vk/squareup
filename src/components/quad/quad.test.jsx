import React from 'react';
import renderer from 'react-test-renderer';

import Quad from './quad';
import Game from '../../game/game';

test('renders correctly', () => {
  const game = new Game();
  const component = renderer
    .create(<Quad game={game} />)
    .toJSON();
  expect(component).toMatchSnapshot();
});

