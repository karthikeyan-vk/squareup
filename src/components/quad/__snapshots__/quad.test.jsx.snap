// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quad"
>
  <img
    src="bg.png"
  />
  <div
    className="squares"
  >
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
    <div
      className="square empty destination"
      onClick={[Function]}
    >
      <span
        className="number"
      />
    </div>
  </div>
</div>
`;
